nerdlets/pathpoint-nerdlet/services/DataManager.js

```diff
@@ -22,6 +22,74 @@ import LogConnector from './LogsConnector';
 import SynConnector from './SynConnector';
 import CredentialConnector from './CredentialConnector';
 
+
+export function TimeRangeTransform(pointInTime, sinceClause) {
```

Comment:
This new version of TimeRangeTransfom takes two parameters...

1. The point in time
2. Part of the since clause.